components/02-molecules/pull-quote/pull-quote.twig

```diff
@@ -0,0 +1,20 @@
+{#
+ # Available Props:
+ # - pull_quote__style: bar-left, bar-right, or quote-left
+ #
+ # Available Variables:
+ # - pull_quote__quote (required)
+ # - pull_quote__attribution
+ #}
+
+{% set pull_quote__base_class = 'pull-quote' %}
+{% set pull_quote__style = pull_quote__style|default('bar-left') %}
+
+<figure {{ bem(pull_quote__base_class) }} data-pull-quote-style="{{ pull_quote__style }}">
```

Comment:
Similar feedback I had for the links. We should use the `add_attributes(pull_quote_attributes)` implementation in case this needs to be extended during implementation. https://github.com/yalesites-org/component-library-twig/pull/10/files#diff-e4131c9c0a0500830af2795a53a11107872df87c88b69e073bf5c396236206fcR17

Reply:
Good idea. I've actually not really used the add_attributes function much in the past. But I think I like the pattern for the things we've done so far on this project. I'll try to remember to do it by default in the future!